start.js

```diff
@@ -20,7 +20,6 @@ function start() {
 }
 
 // lol @ Infinity actually being used for something meaningful
-throng(start, {
-  workers,
-  lifetime: Infinity
-});
+throng({
```

Comment:
the first param doesn't need to be an object if it just specifies the worker count